ocrd_segment/ocrd-tool.json

```diff
@@ -1,11 +1,11 @@
 {
   "version": "0.0.1",
-  "git_url": "https://github.com/OCR-D/ocrd_evaluate_segmentation",
+  "git_url": "https://github.com/OCR-D/ocrd_segment",
   "tools": {
-    "ocrd-evaluate-segmentation": {
-      "executable": "ocrd-evaluate-segmentation",
+    "ocrd-segment-repair": {
```

Comment:
I like _repair_ – but is this going to be shared with eventual `ocrd-segment-merge` (for aligning multiple segmentations heuristically or via specialised classifiers; which could also be used for repair if aligning one fileGrp with itself)?

Reply:
Maybe. But `merge` is way too restrictive in terms of heuristics I dream of.

Reply:
`align` perhaps? Or are you more inclined towards some sort of _layout post-correction_ tool (incidentally allowing merge/alignment)?